pkg/policies/binary/binary.go

```diff
@@ -172,23 +173,43 @@ func (b Binary) Check(ctx context.Context, c *github.Client, owner,
 		return nil, res.Error2
 	}
 
+	logs := convertLogs(l.logs)
+	pass := res.Score >= checker.MaxResultScore
```

Comment:
Maybe check the if `len(binaryArtifactsFound) > 0` here? That way, this code will be agnostic to Scorecard's scoring mechanism.

Reply:
Is that available through checker.CheckResult ?

I think we should take a look at this in #21 and #114.